tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py

```diff
@@ -138,7 +147,7 @@ def __init__(self, dut):
         :param dut: DUT object representing a SONiC switch under test.
         """
         self.dut = dut
-        #self.unlink_file_list = {}
+        # self.unlink_file_list = {}
```

Comment:
can you clean this code?

Reply:
This modification was the update of pre-commit check. My code update does not contain this part, but it failed at pre-commit process.

Reply:
All the codes I need to update was in the first commit. And other code changes are all for passing through the pre-commit check. There are many old codes were not checked by pre-commit then, but when I modified some parts of the old code, pre-commit would check the old codes too, so I edit them in order to pass the check.